Makefile

```diff
@@ -1,11 +1,11 @@
-all: .restart-cqf-ruler .run-load-script .update-cqf-ruler-image
+all: .restart-cqf-ruler .update-cqf-ruler-image
 
 info:
 	$(info usage: `make MEASURE_DIR=/path/to/measure/dir VERSION=x.y.z)
 
 .seed-cqf-ruler-no-vs:
-	cd $(MEASURE_DIR)
-	make connectathon .seed-cqf-ruler-no-vs
+	make -C $(MEASURE_DIR) connectathon .seed-cqf-ruler-no-vs
```

Comment:
If `MEASURE_DIR` isn't specified, this gives a slightly confusing error about the `connectathon` directory not existing.  Perhaps we should add a target that validates that that parameter has been defined.  Could also be part of @tstrass 's DRY out makefiles effort.

`.seed-cqf-ruler-no-vs` only exists for `EXM_105` currently, but since the other measures were moved over here, they will need to have that target added.  I think it makes sense to add them along with this change rather than putting it off for the DRY out in order to keep everything in a working state.

Reply:
@mgramigna this can be part of https://jira.mitre.org/browse/TACOSTRAT-346 unless you'd rather address Luke's comment in this PR. LMK

Reply:
I'm addressing the second paragraph, but will probably rope the first paragraph about `MEASURE_DIR` into 346, yeah